common/customizable_localization/wc_lifestyles_custom_loc.txt

```diff
@@ -387,3 +387,106 @@ GetWCMagicalLifestyleElementalEarthMagicNextTraitForPerks = {
 		localization_key = "wc_elemental_earth_magic_next_trait_level_is_1"
```

Comment:
This branch has a lot of errors.
A few of them:
```
Line 2149: Unknown trigger type: wc_can_start_new_lifestyle_tree_duty_trigger, near line: 10
	Line 2150: Unknown trigger type: wc_can_start_new_lifestyle_tree_skulduggery_trigger, near line: 11
	Line 2151: Unknown trigger type: wc_can_start_new_lifestyle_tree_temptation_trigger, near line: 12
	Line 2152: Unknown trigger type: wc_can_start_new_lifestyle_tree_intimidation_trigger, near line: 13
	Line 2153: Unknown trigger type: wc_can_start_new_lifestyle_tree_medicine_trigger, near line: 14
	Line 2154: Unknown trigger type: wc_can_start_new_lifestyle_tree_scholarship_trigger, near line: 15
	Line 2155: Unknown trigger type: wc_can_start_new_lifestyle_tree_theology_trigger, near line: 16
	Line 2156: " in file: " file: common/lifestyle_perks/wc_shadow_magic_perks.txt line: 457; can_start_new_lifestyle_tree_trigger" near line: 42
```

This branch is to be merged to `magic_trees` and `magic_trees` is to be merged to `master`. Right now `magic_trees` has 18k changed lines. I'm pretty sure I won't check due to its size at this point. How did these errors get there? As the base and core branch of this entire feature, it must be free of such errors. Fixing them must be your top priority before requesting my reviews on child branches.

Reply:
I already was answering on this question multiple times

Reply:
Was fixed in #757 by cherry-picking fixes from #633